### ISSUE TITLE
Tree Chart Link Options

### ISSUE THREAD

**Issue by btbritz** [amcharts/amcharts5#1455]
Is there any way to specify the curvature of a link between nodes to produce something similar to:

<img width="998" alt="Screenshot 2024-04-11 at 9 33 58 AM" src="https://github.com/amcharts/amcharts5/assets/3333099/63c73006-b4d3-431d-8439-8080975de83c">

If not, is there a way to fake it to get a similar effect?

**Comment by martynasma** [comment]
No, I'm afraid. This type of linkage is not supported, and I'm not aware of any workarounds that could be reliably used to "fake" it 🤔
